docs/installation/ubuntulinux.md

```diff
@@ -12,7 +13,9 @@ parent = "smn_linux"
 
 Docker is supported on these Ubuntu operating systems:
 
+- Ubuntu Wily 15.04
 - Ubuntu Vivid 15.04
+- Ubuntu Utopic 14.10
```

Comment:
We should keep an eye on 15.04 as well, which will be EOLd in January (https://wiki.ubuntu.com/Releases)